README.md

```diff
@@ -65,9 +105,11 @@ Spinal Cord MRI (T2 weighted, T2 star, MT) preprocessing include number of key s
     sct_register_to_template -i ${file_t2w}.nii.gz -s ${file_t2_seg}.nii.gz -ldisc ${file_t2_labels_discs}.nii.gz -c t2 -qc qc
 ```
 ## Quality Control:
-- After preprocessing, perform a QC check by reviewing the HTML files in the QC directory.
-- Inspect the T2-weighted and T2-star images for segmentation and vertebral level labeling errors.
-- If errors (e.g., segmentation leakage or under-segmentation) and/or labelling error are found, manually correct them.
+- After preprocessing, perform a QC check by reviewing the HTML files in the QC directory: `<path-out>/qc/index.html`.
+- Inspect the T2-weighted and T2-star images for segmentation and vertebral level labeling errors:
+<ADD EXAMPLE>
```

Comment:
@Mfauziyya you could add an example of segmentation error
How to do the correction? 
What needs to be done for disc labels vs for segmentation